Mathlib/AlgebraicGeometry/AffineSpace.lean

```diff
@@ -461,6 +461,14 @@ lemma isIntegralHom_over_iff_isEmpty : IsIntegralHom (𝔸(n; S) ↘ S) ↔ IsEm
 lemma not_isIntegralHom [Nonempty S] [Nonempty n] : ¬ IsIntegralHom (𝔸(n; S) ↘ S) := by
   simp [isIntegralHom_over_iff_isEmpty]
 
+/--
+One should bare this equality in mind when breaking the `Spec R/ PrimeSpectrum R` abstraction
+boundary.
+-/
+lemma preorder_instance_eq_spec (R : CommRingCat) :
+  inferInstance (i := PartialOrder (Spec R)) =
+  inferInstance (i := PartialOrder (PrimeSpectrum R)) := by rfl
```

Comment:
I'm not clear what the intention is here but I doubt this is saying what you intend. For example:
```lean
example {α : Type*} {i : α} : inferInstance (i := i) = i := rfl
```
so all you're really saying is:
```lean
example : PartialOrder (Spec R) = PartialOrder (PrimeSpectrum R) := rfl
```
which of course is true since:
```lean
example : Spec R = PrimeSpectrum R := rfl
```
Possibly you want to write a lemma like:
```lean
example (R : CommRingCat) (p q : Spec R) :
    p ≤ q ↔ p.asIdeal ≤ q.asIdeal :=
```